<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTorrentDialog</class>
 <widget class="QDialog" name="AddTorrentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>353</height>
   </rect>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0">
     <item row="2" column="1">
      <widget class="QLabel" name="titleExampleLabel">
       <property name="text">
        <string>Examples:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="titleListLabel">
       <property name="font">
        <font>
         <family>Courier New</family>
        </font>
       </property>
       <property name="text">
        <string notr="true">   C:\User\My Documents\hello_world.torrent
   magnet:?xt=urn:btih:0123456789abcdef0123456789abcdef&amp;dn=Hello+World</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="urlLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Download:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Note: a magnet link contains info-hash metadata to download the .torrent file. If a magnet link is given, the application downloads the .torrent to your system's temporary directory, and thus adds it to the queue (started or paused).</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="titleLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Enter the .torrent file (or magnet link) to download</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="UrlLineEdit" name="urlLineEdit"/>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Magnet Link and Torrent</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="3">
      <widget class="QWidget" name="logo" native="true">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/icons/menu/preference_torrent_32x32.png);</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="UrlFormWidget" name="urlFormWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>100</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonBox" stretch="1,0,0,0">
     <property name="spacing">
      <number>9</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Start!</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addPausedButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Add &amp;paused</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UrlLineEdit</class>
   <extends>QLineEdit</extends>
   <header>Widgets/UrlLineEdit</header>
  </customwidget>
  <customwidget>
   <class>UrlFormWidget</class>
   <extends>QWidget</extends>
   <header>Widgets/UrlFormWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>urlLineEdit</tabstop>
  <tabstop>startButton</tabstop>
  <tabstop>addPausedButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>startButton</sender>
   <signal>released()</signal>
   <receiver>AddTorrentDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>509</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addPausedButton</sender>
   <signal>released()</signal>
   <receiver>AddTorrentDialog</receiver>
   <slot>acceptPaused()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>762</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>644</x>
     <y>509</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>released()</signal>
   <receiver>AddTorrentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>846</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>762</x>
     <y>509</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>acceptPaused()</slot>
 </slots>
</ui>
